packages/webapp/src/routes/__tests__/credits-purchase/check-and-submit.spec.js

```diff
@@ -13,6 +13,12 @@ const auth = {
   }
 }
 
+const populateSessionWithDevelopmentProjectInfo = (session) => {
+  session.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_DEVELOPMENT_NAME, 'Anything')
+  session.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_APPLICATION_REFERENCE, 'BNGCRD-L4XCQ-AIZMO')
```

Comment:
This should be `CREDITS_PURCHASE_PLANNING_APPLICATION_REF` for the planning application reference